config/matomoWalletsEvents.ts

```diff
@@ -0,0 +1,332 @@
+import { MatomoEventType, trackEvent } from '@lidofinance/analytics-matomo';
+import { Metrics as WalletsMetrics } from 'reef-knot';
+
+const MATOMO_EVENT_CATEGORY = 'Lido_Frontend_Template';
```

Comment:
Maybe add a comment to make it clear, that this line should be changed to the name of Lido on X project?

Reply:
Good idea 👍